scss/_tables.scss

```diff
@@ -32,3 +32,7 @@ th {
   padding: .825 * $em $em;
 }
 
+thead th {
+  border-bottom-width: 2px;
```

Comment:
yes, sure. we should use only `$em`, never `px`. if you see `px` anywhere, please replace them